playbooks/openshift-master/private/config.yml

```diff
@@ -192,6 +206,17 @@
     openshift_master_default_registry_value: "{{ hostvars[groups.oo_first_master.0].l_default_registry_value }}"
     openshift_master_default_registry_value_api: "{{ hostvars[groups.oo_first_master.0].l_default_registry_value_api }}"
     openshift_master_default_registry_value_controllers: "{{ hostvars[groups.oo_first_master.0].l_default_registry_value_controllers }}"
+    when: not ( openshift_master_bootstrap_enabled | default(false) | bool )
+
+  - role: openshift_control_plane
+    openshift_master_hosts: "{{ groups.oo_masters_to_config }}"
+    r_openshift_master_clean_install: "{{ hostvars[groups.oo_first_master.0].l_clean_install }}"
+    r_openshift_master_etcd3_storage: "{{ hostvars[groups.oo_first_master.0].l_etcd3_enabled }}"
```

Comment:
task 'Determine if etcd3 storage is in use' above sets l_etcd3_enabled.  This appears it will never be true on a new cluster install because it's grepping for etcd3 in master's config.

Reply:
it should always be true after 3.7, so i'm going to force it to true here